### PR TITLE
chore: add component playground to storybook

### PR DESCRIPTION
**Pull request by dcwarwick**
Add a component playground into the storybook. This shows multiple components working together.

#### What did you change?

Added a story to the storybook

#### How did you test and verify your work?

Ran storybook

### CODE REVIEW

File: packages/core/src/component-playground/ComponentPlayground.stories.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { getPathForComponent } from '../../story-structure';
+
+import App from './ComponentPlayground';
+
+export default {
+  title: getPathForComponent('o', 'Examples'),
```
Comment (lee-chase) [inline packages/core/src/component-playground/ComponentPlayground.stories.js:14]: Change layout to full screen 

`
  parameters: { layout: 'fullscreen' },
`

**Review by lee-chase — CHANGES_REQUESTED** [review]
Looks good, 1 change request to make fullscreen otherwise the header looks odd.

- It would also be nice if there was no content jump on showing the side panel. 
- Personally prefer the scroll bar to be under the global header as per in-context example.

File: packages/core/src/component-playground/ComponentPlayground.stories.js (hunk 1 above)
Comment (dcwarwick) [reply packages/core/src/component-playground/ComponentPlayground.stories.js:14]: Good catch -- will check that in. @matthewgallo This fixes the extra space we were seeing around the PageHeader, including the content showing through.

Comment (matthewgallo) [reply packages/core/src/component-playground/ComponentPlayground.stories.js:15]: I jumped in to help with some things, including adding this piece ^

**Pull request merged**